### ISSUE TITLE
Changes to the deployment is not reflecting in the git hub page for more than 4 hours.

### ISSUE THREAD

**Issue by diwakar-ravichandran** [alshedivat/al-folio#70]
The repo shows the right versions and contains all of the content to be hosted by the github page. But it does not reflect in the github page itself. I have tried all the different solutions that were mentioned by different sources. None of them seem to help me. Can some one help me out here?

**Comment by alshedivat** [comment]
Sorry, I don't understand the question. If you run `bin/deploy.sh`, it should correctly deploy your website on gihub pages. Note that in your fork of the repository in settings you need to point github pages to the branch you deployed the website (`gh-pages` or `master`).

**Comment by diwakar-ravichandran** [comment]
@alshedivat Thanks for the response. I sort of figured out the problem. It was a case of double publishing. So my website was getting deployed from the fork that I made of yours. And also from the github.io repo that I made. So this caused multiple deployments to the same page. Later I realized the deployments came from the forked repo and not the github.io repo. Thanks again for following up.
